core/controller/src/whisk/core/controller/Actions.scala

```diff
@@ -456,19 +475,19 @@ trait WhiskActionsApi extends WhiskCollectionAPI {
      * either there is an error in the get, or the promise has completed because it timed out. The promise MUST
      * complete in the caller to terminate the polling.
      */
-    private def pollDbForResult(docid: DocInfo, activationId: ActivationId, resultOnly: Boolean,
-                                promise: Promise[(ActivationId, Option[(ActivationResponse, JsObject)])])(implicit transid: TransactionId): Unit = {
+    private def pollDbForResult(
+        docid: DocInfo,
+        activationId: ActivationId,
+        promise: Promise[Option[WhiskActivation]])(
+            implicit transid: TransactionId): Unit = {
         if (!promise.isCompleted) {
             WhiskActivation.get(activationStore, docid) map {
-                fullActivation =>
-                    val activation = fullActivation.stripLogs
```

Comment:
So that we are consistent about what gets returned regardless of which path was taken.